config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -1921,10 +1921,6 @@
 # Default: false
 # gradebook.coursegrade.displayed=true
 
-# Default for course grade calculated
-# Default: true
-# gradebook.coursegrade.showCalculatedGrade=false
```

Comment:
Please, don't remove this property.

Reply:
This PR reverts the two JIRAs. Please provide some feedback on the issues identified and how they can be *quickly* fixed. If they can't be fixed quickly, the two commits should be reverted until they can be fixed.